src/utils/transform/cache.ts

```diff
@@ -8,6 +8,20 @@ const getTime = () => Math.floor(Date.now() / 1e8);
 
 const tmpdir = os.tmpdir();
 const noop = () => {};
+
+/**
+ * Cache directory name is based on the user's uid.
+ * This is to avoid permission issues when as a different user in parallel.
+ */
+function cacheDirectoryName(): string {
+	const { geteuid } = process;
+	if (geteuid === undefined) {
```

Comment:
I realized this might not exist on Windows.

I'm currently refactoring the tests to be lighter so I can bring back Windows. Will hold this until then.